test/models/MongoDateTimeFormatsSpec.scala

```diff
@@ -24,10 +24,10 @@ class MongoDateTimeFormatsSpec extends SpecBase {
 
   "MongoDateTimeFormats" should {
 
-    //TODO - Result mismatch of res & testReads
-    "read" ignore new Setup {
-      val res = MongoDateTimeFormats.localDateTimeRead
-      res mustBe testReads
+    "read" in new Setup {
+      val testData: JsValue = Json.parse(""" {"$date": 1704228313308}  """)
+      val res = MongoDateTimeFormats.localDateTimeRead.reads(testData)
+      res.isSuccess mustBe true
```

Comment:
Better to give a line break before this line to differentiate the assertion and to make it more readable